app/components/EventItem/styles.css

```
@@ -1,11 +1,18 @@
 .eventItem {
   min-height: 50px;
+  padding: var(--spacing-xs);
   padding-left: var(--spacing-md);
   border-left: 4px solid transparent;
   margin-bottom: var(--spacing-md);
   display: flex;
   justify-content: space-between;
   line-height: 1.3;
+  transition: var(--easing-fast);
+}
+
+.eventItem:hover {
+  background-color: var(--additive-background);
+  border-radius: var(--border-radius-sm);
 }
```

```suggestion

  &:hover {
    background-color: var(--additive-background);
    border-radius: var(--border-radius-sm);
  }
}
```
Use `&:hover` nested inside the class to add hover styling to the same classname:)